### ISSUE TITLE
validation fails for 4844 csgo and lol map state in handle_state_message

### ISSUE THREAD

**Issue by sf-chris** [Sportsflare/bass-tube#18]
## TLDR
lol and csgo map state trying to be handled in `handle_state_message` see  https://sentry.io/organizations/sportsflare/issues/3569307244/?project=6261002&query=is%3Aunresolved#message

## Frontend issue
[54125 wont close](https://sportsflare-houston.web.app/dashboard/markets/?id=54125) 
BT_ID=4844 DETAILS_ID=54125

## Sentry issues

https://sentry.io/organizations/sportsflare/issues/3569326148/?project=6261002&query=is%3Aunresolved#exception
https://sentry.io/organizations/sportsflare/issues/3569331858/?project=6261002&query=is%3Aunresolved
https://sentry.io/organizations/sportsflare/issues/3569308174/?project=6261002&query=is%3Aunresolved
https://sentry.io/organizations/sportsflare/issues/3569307244/?project=6261002&query=is%3Aunresolved
https://sentry.io/organizations/sportsflare/issues/3569307352/?project=6261002&query=is%3Aunresolved

*Originally created on GitLab by leo_sf (https://gitlab.com/sportsflare/real-time/bass-tube/-/issues/58)*

**Comment by sf-chris** [comment]
Lost a bit of context here with issues disappearing from Sentry. I think we can ignore this until it happens again.

*Originally commented on GitLab by steffansafey*

**Comment by sf-chris** [comment]
no reference to 4988 occurs in the kafka logs for 4844 that could have slipped through. AFAICT the could arise if the fixture returned by getting the fixture_id from the message on https://gitlab.com/sportsflare/real-time/bass-tube/-/blob/c4206ab3dd6af85c6faeffd14753053c8974ee8e/bass_tube/state_ingest/handlers/state/__init__.py#L20 was a csgo fixture, and then the "correct" lol state for map one got stitched into the `existing_maps` object. As to how this could occur, thats where Im stumped.

*Originally commented on GitLab by leo_sf*

**Comment by sf-chris** [comment]
Sentry issue: [BASS-TUBE-API-9](https://sentry.io/organizations/sportsflare/issues/3569307244/?referrer=gitlab_integration)

*Originally commented on GitLab by sfsentry1*

**Comment by sf-chris** [comment]
Sentry issue: [BASS-TUBE-API-A](https://sentry.io/organizations/sportsflare/issues/3569307352/?referrer=gitlab_integration)

*Originally commented on GitLab by sfsentry1*

**Comment by sf-chris** [comment]
Sentry issue: [BASS-TUBE-API-B](https://sentry.io/organizations/sportsflare/issues/3569308174/?referrer=gitlab_integration)

*Originally commented on GitLab by sfsentry1*

**Comment by sf-chris** [comment]
Sentry issue: [BASS-TUBE-API-C](https://sentry.io/organizations/sportsflare/issues/3569326148/?referrer=gitlab_integration)

*Originally commented on GitLab by sfsentry1*

**Comment by sf-chris** [comment]
Sentry issue: [BASS-TUBE-API-D](https://sentry.io/organizations/sportsflare/issues/3569331858/?referrer=gitlab_integration)

*Originally commented on GitLab by sfsentry1*

**Comment by sf-chris** [comment]
:shrug: 

im open to other big brains weighing in, but unless it happens again, low prio.

*Originally commented on GitLab by leo_sf*

**Comment by sf-chris** [comment]
looking at the kafka logs, there doesn't appear to be any map 2 states that get passed through.

However, when looking at https://sentry.io/organizations/sportsflare/issues/3569307244/?project=6261002&query=is%3Aunresolved#message

in the stack for `handle_state_message`,  `existing_maps` has both a lol map state for map 1 (which looks to reference the correct participants), and a csgo map state for map 2 which references participants `4626` "Dynamo eclot" and `4146` "Jano".

This match was [DT=54179](https://sportsflare-houston.web.app/dashboard/settling/?id=54179) and BT=4898. and was in the Elisa Invitational on the 5th of september.

*Originally commented on GitLab by leo_sf*

**Comment by sf-chris** [comment]
Basstube throws a 500, details works fine.

results are {}, state is got an odd map 2 for a BO1. All other games in that fixture are BO1 and dont appear to have the same issue.

Set state to null in the db and closed the fixture

*Originally commented on GitLab by leo_sf*
